host/frontend/host-orchestrator/instancemanager_test.go

```diff
@@ -102,53 +102,79 @@ func (d *FakeFetchCVDDownloader) Download(dst io.Writer, buildID string) error {
 	return nil
 }
 
-func TestFetchCVDHandler(t *testing.T) {
+func TestFetchCVDHandlerDownloadBinaryAlreadyExist(t *testing.T) {
 	dir := t.TempDir()
 	f, err := os.Create(BuildFetchCVDFileName(dir, "1"))
 	if err != nil {
 		t.Fatal(err)
 	}
 	defer f.Close()
-	_, err = f.Write([]byte(string("000")))
+	_, err = f.Write([]byte(string("bar")))
 	if err != nil {
 		t.Fatal(err)
 	}
-	downloader := &FakeFetchCVDDownloader{
-		t:       t,
-		content: "111",
+	downloader := &FakeFetchCVDDownloader{t, "foo"}
+	h := NewFetchCVDHandler(dir, downloader)
+
+	err = h.Download("1")
+
+	if err != nil {
+		t.Errorf("epected <<nil>> error, got %#v", err)
+	}
+	content, err := ioutil.ReadFile(BuildFetchCVDFileName(dir, "1"))
+	if err != nil {
+		t.Fatal(err)
+	}
+	actual := string(content)
+	expected := "bar"
```

Comment:
please use a variable or constant local to this function for "bar" here and in line 112

Reply:
done